resources/data/ses_config-0.1.2.json

```diff
@@ -43,15 +43,20 @@
             "steps": [
                 {
                     "type": "Step",
-                    "title": "Choose paths",
+                    "title": "Import project",
                     "description": [
                         "Select **File > Open nRF Connect SDK Project**.",
                         "To import a project into SES, you must specify the following information:",
-                        "- **CMakeLists.txt** - the location of the ``CMakeLists.txt`` project file of the sample that you want to work with (for example, the full path to ``ncs/nrf/applications/asset_tracker/CMakeLists.txt``)",
-                        "- **Board Directory** - the location of the board description for which to build the project (for example, the full path to ``ncs/zephyr/boards/arm/nrf9160_pca10090``)",
-                        "- **Board Name** - the board name for which to build the project (for example, ``nrf9160_pca10090ns``)",
-                        "- **Build Directory** - the folder in which to run the build (for example, the full path to ``ncs/nrf/applications/asset_tracker/build``)",
-                        "- **Delete Existing CMakeCache.txt** - select this option to ensure that you are not building with an outdated build cache",
+                        "- **nRF Connect SDK Release** - Select the nRF Connect SDK version that you want to work with.",
+                        "- **nRF Connect Toolchain Version** - Select NONE and make sure that your SES environment is configured correctly.",
```

Comment:
Can you have a link to the docs here? I think I would be confused if I'm only being told to check if my environment is configured correctly, but not how it should be configured.
In the docs, we have "Select NONE and make sure that your SES environment is configured correctly (see Setting up the SES environment)." with link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#setting-up-ses

Reply:
Yes, however that is part of the previous step. So could this be even simpler?

Reply:
Sure, if we know this is configured correctly, it's sufficient to just say "Select NONE."

Reply:
It's probably also the only option in the drop-down in this case, isn't it?